.devcontainer/advanced-integration/welcome-message.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+WELCOME_MESSAGE="
+👋 Welcome to the \"Advanced Checkout Integration Example\"
+
+🛠️  Your environment is fully setup with all the required software.
+
+🚀 Once you rename the \".env.example\" file to \".env\" and update \"CLIENT_ID\" and \"APP_SECRET\", the checkout page will automatically open in the browser after the server is restarted."
+
+ALTERNATE_WELCOME_MESSAGE="
+👋 Welcome to the \"Advanced Checkout Integration Example\"
+
+🛠️  Your environment is fully setup with all the required software.
+
+🚀 The checkout page will automatically open in the browser after the server is started."
+
+if [ -n "$CLIENT_ID" ] && [ -n "$APP_SECRET" ]; then
+    WELCOME_MESSAGE="${ALTERNATE_WELCOME_MESSAGE}"
+fi
+
+sudo bash -c "echo \"${WELCOME_MESSAGE}\" > /usr/local/etc/vscode-dev-containers/first-run-notice.txt"
```

Comment:
Without sudo, we can't update file in /usr/local/etc/vscode-dev-containers. After Dev environment is setup, it will get the message from first-run-notice.txt and display it on the terminal
<img width="1715" alt="image" src="https://github.com/paypal-examples/docs-examples/assets/94279589/d1918266-2973-4285-b8e0-4425b2e70365">

Reply:
Without sudo, we can't update file in /usr/local/etc/vscode-dev-containers. After Dev environment is setup, it will get the message from first-run-notice.txt and display it on the terminal